localstack/services/awslambda/invocation/lambda_models.py

```diff
@@ -433,12 +433,9 @@ class ResourcePolicy:
 
 @dataclasses.dataclass
 class FunctionResourcePolicy:
-    revision_id: str = dataclasses.field(init=False, default_factory=long_uid)
-    policy: ResourcePolicy  # TODO: do we have a typed IAM policy somewhere already?
-
-    @staticmethod
-    def new_revision_id() -> str:
-        return long_uid()
+    # TODO: do we have a typed IAM policy somewhere already?
+    # Something like this? localstack_ext.services.iam.policy_engine.models.PolicyDocument
```

Comment:
```suggestion
```

I'd just remove this for now. There's not too much that we can do with it anyway.